Submitted By: Ken Moffat <ken at linuxfromscratch dot org>
Date: 2013-10-09
Initial Package Version: 0.5.1 but only needed on LFS-7.3.
Upstream Status: Unknown
Origin: Arch, AUR user 'oshazard' http://code.google.com/p/aur-includes/source/browse/pnmixer-fix/pnmixer-0.5.1-configure.in.patch
Description: Works around the following error when ./autogen.sh is run:

./configure: line 1867: syntax error near unexpected token `pnmixer,'
./configure: line 1867: `AM_INIT_AUTOMAKE(pnmixer, 0.5.1)'

 That is presumably an error in autotools which has been fixed in the current versions.

 Rediffed to apply with -p1

--- pnmixer-0.5.1/configure.in.orig	2011-12-18 23:34:56.000000000 +0000
+++ pnmixer-0.5.1/configure.in	2013-10-07 16:57:40.894083484 +0100
@@ -1,14 +1,12 @@
 dnl Process this file with autoconf to produce a configure script.
 
-AC_INIT(configure.in)
-AM_INIT_AUTOMAKE(pnmixer, 0.5.1)
-AM_CONFIG_HEADER(config.h)
+AC_INIT([pnmixer], [0.5.1])
+AM_INIT_AUTOMAKE
 AM_MAINTAINER_MODE
 
 OLD_CFLAGS=$CFLAGS
 AC_ISC_POSIX
 AC_PROG_CC
-AM_PROG_CC_STDC
 AC_HEADER_STDC
 CFLAGS=$OLD_CXXFLAGS
 
